tests/Date/DateMutabilityConversionTest.php

```diff
@@ -10,10 +10,12 @@
  * @link          http://cakephp.org CakePHP(tm) Project
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
+namespace Cake\Chronos\Test\Date;
+
 use Cake\Chronos\Date;
 use Cake\Chronos\MutableDate;
 
-class DateMutabilityConversionTest extends TestCase
+class DateMutabilityConversionTest extends \TestCase
```

Comment:
I updated the PR. But the build is failing and I suspect it's not because of my changes:

```
1) Cake\Chronos\Test\DateTime\DiffTest::testDiffForHumansRelative with data set #3 (Cake\Chronos\Chronos Object (...), Cake\Chronos\Chronos Object (...), '1 month before')
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
-1 month before
+4 weeks before
```